### PR TITLE
added support for gaussian likelihood

### PR DESCRIPTION
**Pull request by davidsebfischer**
added support for gaussian likelihood

### CODE REVIEW

File: sfaira/models/embedding/output_layers.py [hunk 1]
```diff
@@ -91,7 +91,116 @@ def call(self, inputs, **kwargs):
...
+    """
+    Gaussian output layer.
+
+    Size factor only makes sense if logged and data is positive and logged.
+    """
+
+    def __init__(
+            self,
+            original_dim=None,
+            name='neg_bin_output',
```
Comment (le-ander) [inline sfaira/models/embedding/output_layers.py:110]: @davidsebfischer did you on purpose not change the name of the output layer here?

File: sfaira/models/embedding/output_layers.py [hunk 2]
```diff
@@ -91,7 +91,116 @@ def call(self, inputs, **kwargs):
...
+    """
+    Gaussian output layer with a single standard deviation estimate per features.
+
+    Size factor only makes sense if logged and data is positive and logged.
+    """
+
+    def __init__(
+            self,
+            original_dim=None,
+            name='neg_bin_shared_disp_output',
```
Comment (le-ander) [inline sfaira/models/embedding/output_layers.py:144]: @davidsebfischer did you on purpose not change the name of the output layer here?

File: sfaira/models/embedding/output_layers.py [hunk 3]
```diff
@@ -91,7 +91,116 @@ def call(self, inputs, **kwargs):
...
+    """
+    Gaussian output layer with standard deviation set as constant (=1).
+
+    Size factor only makes sense if logged and data is positive and logged.
+    """
+
+    def __init__(
+            self,
+            original_dim=None,
+            name='neg_bin_const_disp_output',
```
Comment (le-ander) [inline sfaira/models/embedding/output_layers.py:183]: @davidsebfischer did you on purpose not change the name of the output layer here?

File: sfaira/models/embedding/output_layers.py (hunk 1 above)
Comment (davidsebfischer) [reply sfaira/models/embedding/output_layers.py:110]: no, will change

**Pull request merged**